content/applications/finance/payment_acquirers/ogone.rst

```diff
@@ -5,84 +5,107 @@ Ogone
+-----------------
+
+Create an API user
+~~~~~~~~~~~~~~~~~~
+
+Log into your Ogone account and head to the :guilabel:`Configuration` tab.
+
+You need to create an **API user** to be used in the creation of transactions from Odoo. While you
+can use your main account to do so, using an **API user** ensures that if the credentials used in
+Odoo are leaked, no access to your Ogone configuration is possible. Additionally, passwords for
+**API users** do not need to be updated regularly, unlike normal users.
+
+To create an **API user**, go to :menuselection:`Configuration --> Users` and click on
+:guilabel:`New User`. The following fields must be configured:
+
+.. _ogone/ogone:
+
+- :guilabel:`UserID`: you can choose anything you want.
+- :guilabel:`User's Name, E-mail and Timezone`: you can enter the information you want.
+- :guilabel:`Profile`: should be set to :guilabel:`Admin`.
+- :guilabel:`Special user for API`: should be checked.
+
+After the creation of the user, you are required to generate a password. Save the password and
+**UserID**, as they will be required later on during the setup.
+
+Set up Ogone for Odoo
+~~~~~~~~~~~~~~~~~~~~~
+
+Ogone must now be configured to accept payments from Odoo. Head to :menuselection:`Configuration -->
+Technical Information --> Global Security Parameters`, select :guilabel:`SHA-1` as
+:guilabel:`Hash Algorithm` and :guilabel:`UTF-8` as :guilabel:`character encoding`. Then, go to the
+:guilabel:`Data and Origin Information` tab of the same page and leave the URL field of the
+:guilabel:`e-Commerce and Alias Gateway` section blank.
+
+.. tip::
+   If you need to use another algorithm, such as `sha-256` or `sha-512`, within Odoo, activate the
+   :ref:`developer mode <developer-mode>` and go to :menuselection:`General Settings -->
+   Technical --> System Parameters`. From here, search for `payment_ogone.hash.function` and change
+   the value line to the desired algorithm (`sha-256` or `sha-512`).
+
+You are now required to generate **SHA-IN** passphrases. **SHA-IN** and **SHA-OUT** passphrases are
+used to digitally sign the transaction requests and responses between Odoo and Ogone. By using these
+secret passphrases and the `sha-1` algorithm, both systems can ensure that the information they
+receive from the other was not altered or tampered with.
+
+Enter the same **SHA-IN** passphrase in both :guilabel:`Checks for e-Commerce & Alias Gateway` and
+:guilabel:`Checks for DirectLink and Batch (Automatic)`. You can leave the IP address field blank.
+
+Your **SHA-IN** and **SHA-OUT** passphrases should be different, and between 16 and 32 characters
+long. Make sure to use the same **SHA-IN** and **SHA-OUT** passphrases throughout the entire Ogone
+configuration, as Odoo only allows a single **SHA-IN** and single **SHA-OUT** passphrase.
+
+In order to retrieve the **SHA-OUT** key, log into your Ogone account, go to
+:menuselection:`Configuration --> Technical --> Information --> Transaction feedback --> All
```

Comment:
It should be Configuration --> Technical Information --> Transaction feedback ...